apps/storybook-react/.storybook/main.js

```diff
@@ -1,7 +1,13 @@
+const path = require('path')
+
 module.exports = {
-  // typescript: {
-  //   reactDocgen: 'react-docgen',
```

Comment:
remove dead code?

Reply:
Not sure what you mean, it’s already removed

Reply:
I need more coffee ☕ 🙈